### PR TITLE
automl: video beta move model samples from branch to master

### PR DESCRIPTION
**Pull request by nnegrey**

### CODE REVIEW

File: automl/beta/get_model_evaluation_test.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+from google.cloud import automl_v1beta1 as automl
+import pytest
+
+import get_model_evaluation
+
+PROJECT_ID = os.environ["AUTOML_PROJECT_ID"]
+MODEL_ID = os.environ["ENTITY_EXTRACTION_MODEL_ID"]
+
+
+@pytest.fixture(scope="function")
```
Comment (leahecole) [inline automl/beta/get_model_evaluation_test.py]: I ❤️ fixtures

File: automl/beta/delete_model_test.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+PROJECT_ID = os.environ["AUTOML_PROJECT_ID"]
+
+
+def test_delete_model(capsys):
+    # As model creation can take many hours, instead try to delete a
+    # nonexistent model and confirm that the model was not found, but other
+    # elements of the request were valid.
+    try:
+        delete_model.delete_model(PROJECT_ID, "TRL0000000000000000000")
```
[the remark this answers is not in the document]
Comment (nnegrey) [reply automl/beta/delete_model_test.py]: Nope

**Comment by nnegrey** [comment]
So while these samples are for the video beta, the samples can be used with any automl product. 

The samples that are only for video can be found on this PR and those have the `video_` prefix.
https://github.com/GoogleCloudPlatform/python-docs-samples/pull/2750

**Comment by nnegrey** [comment]
But I did miss some `_beta` on the region tags

**Pull request merged**